### DISCUSSION TITLE
Can't get summary to show up using $GITHUB_STEP_SUMMARY

### DISCUSSION THREAD

**Discussion by RachelAyoroa** [community/community#36092] [Actions]
I have this very simple yaml that I shortened up to remove other steps I'm doing, but in this step I just want to see a summary show up. This yaml is called from a different yaml file and everything works as expected except I never see a summary. I tried single quotes and double quotes around the Hello statement. I do see the download artifact appear (specified on the main yaml that calls this yaml) but I never see a summary. Any idea what I could check? Thanks!
jobs:
  my_job:
    runs-on: [ self-hosted, windows ]
    timeout-minutes: 1500
    steps:
      - name: Adding markdown
        run: echo "### Hello Friends!!" >> $GITHUB_STEP_SUMMARY

**Comment by Meltz014** [comment]
Old thread, but i'm seeing the same issue on a self-hosted runner for a repo hosted in Github Enterprise Server 3.7.4
